reference/faq.md

```diff
@@ -414,7 +414,9 @@ EOF
 
 {{site.prodname}} will then route traffic between {{site.prodname}} hosts using IP-in-IP.
 
-In AWS, you disable `Source/Dest. Check` instead of using IP-in-IP as long as all your instances are in the same subnet of your VPC.  This will provide the best performance.  You can disable this with the CLI, or right click the instance in the EC2 console, and select `Change Source/Dest. Check` from the `Networking` submenu.
+For best performance in AWS, you can disable [Source/Destination Check]({{ site.baseurl }}/reference/resources/felixconfig#spec) instead of using IP-in-IP or VXLAN; but only if all your instances are in the same subnet of your VPC. The setting must be `Disable` for the EC2 instance(s) to process traffic not matching the host interface IP address.
```

Comment:
We should probably mention cross-subnet mode too.

Reply:
@lxpollitt -- I added some details at :417 separate from the highlighted section. I wasn't sure how to fit `CrossSubnet` mode in the highlighted section. Please let me know if you have any suggestions.

Thanks

Reply:
Alex, addressed the comment per discussion. marking resolved but please feel free to reopen.